packages/nimble-components/src/anchor-button/tests/anchor-button.stories.ts

```diff
@@ -0,0 +1,131 @@
+import type { Meta, StoryObj } from '@storybook/html';
+import { withXD } from 'storybook-addon-xd-designs';
+import { html, when } from '@microsoft/fast-element';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+import '../../all-components';
+import {
+    ButtonAppearance,
+    ButtonAppearanceVariant
+} from '../../patterns/button/types';
+
+interface AnchorButtonArgs {
+    label: string;
+    href: string;
+    appearance: string;
+    appearanceVariant: string;
+    icon: boolean;
+    endIcon: boolean;
+    contentHidden: boolean;
+    disabled: boolean;
+}
+
+const appearanceVariantDescription = `
```

Comment:
Are these descriptions identical to the ones for buttons and toggle buttons? Could we try sharing them? One idea would be to add a file to the `patterns/button` directory that exports these `const`s and import them from each story.

If that pattern works we could consider sharing descriptions of link attributes too.

Reply:
Shared the button doc strings. I considered doing it for link attributes, too, but there is only one, and it is only shared between two components (anchor and anchor button). I didn't think it was necessary to create a new file to share one string between two places.